src/components/navigation/document-tab-content.tsx

```diff
@@ -79,7 +79,7 @@ export const DocumentTabContent: React.FC<IProps> = ({ tabSpec }) => {
         <div className={`document-title`}>
           {documentTitle(referenceDocument, appConfigStore, problemStore)}
         </div>
-        {editButton(tabSpec.tab, sectionClass, referenceDocument)}
+        {!referenceDocument.isRemote && editButton(tabSpec.tab, sectionClass, referenceDocument)}
```

Comment:
Think it's worth creating an `isLocal` accessor to avoid the negation in situations like this?

Reply:
I think it's fine for now.  It's likely not going to be needed very often.